pages/add/add.jsx

```diff
@@ -190,12 +140,21 @@ export default React.createClass({
       </div>
     );
 
+
+    let Content;
```

Comment:
we only need to toggle on logged in vs. not logged in, not on staff vs non-staff vs. anonymous